convraw_vid2nwb/convraw_vid2nwb.m

```diff
@@ -0,0 +1,86 @@
+function [nwb] = convraw_vid2nwb(vidfilepath, varargin)
+
+% Example usage:
+% 
+% vidfilepath = 'Y:\\yll\\NMRCNWB\\rawvideofiles';
+% nwb = convraw_vid2nwb(vidfilepath,'nwb_in', nwb, 'animal', 'Barb');

```

Comment:
Please update the examples using the latest version

nwb = convraw_vid2nwb(vidfilepath,'nwb_in', nwb, 'identifier', 'testVideo');